code/style.css

```diff
@@ -1,7 +1,247 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+body {
+  font-family: 'Open Sans Condensed', sans-serif;
+  margin: 0;
+  width: 100%;
+}
 
-.active + .answer {
-  /* some styling here */
+header {
```

Comment:
Really nice header video! I didn´t manage to put a video there. Only a picture. Will try to use your code and see if I can understand where I did wrong. THANK YOU! :-)